Library/Homebrew/descriptions.rb

```diff
@@ -1,8 +1,8 @@
 require "formula"
-require "csv"
+require "formula_versions"
 
 class Descriptions
-  CACHE_FILE = HOMEBREW_CACHE + "desc_cache"
+  CACHE_FILE = HOMEBREW_CACHE + "desc_cache.json"
```

Comment:
For many reasons, (to prevent multiple instance of brew writing cache file, make sure we finish the writing, etc.) We need to use `atomic_write` when saving the cache. And due to lack of `dump` in csv module, I choose to use json. Therefore I have to change the cache filename as  the format isn't compatible. 

>  If so, maybe migrate it?

I don't think we need this as the cost to regenerate cache is considerable small.